.github/workflows/pull-request.yaml

```diff
@@ -29,11 +29,12 @@ jobs:
       - uses: actions/checkout@v2
       - uses: Swatinem/rust-cache@v1
       - run: cargo test --locked --all-features
+      - run: curl -L https://repo1.maven.org/maven2/org/flywaydb/flyway-commandline/7.3.1/flyway-commandline-7.3.1.tar.gz -o flyway-commandline-7.3.1.tar.gz && tar -xzf flyway-commandline-7.3.1.tar.gz --strip-components=1 && rm flyway-commandline-7.3.1.tar.gz
       - run: |
           sudo systemctl start postgresql.service
           sudo -u postgres createuser $USER
           sudo -u postgres createdb $USER
-          psql -f database/schema.sql
+          flyway-commandline-7.3.1/flyway -url="jdbc:postgresql://localhost/?user="$USER"&password=" -locations="filesystem:database/sql/" migrate 
```

Comment:
ubernit: the flyway 7.3.1 is repeated in at least 5 places, maybe an env var could help u change versions easily